src/SixLabors.Fonts/Tables/General/CMapTable.cs

```diff
@@ -12,40 +12,40 @@ namespace SixLabors.Fonts.Tables.General
     internal sealed class CMapTable : Table
     {
         private const string TableName = "cmap";
-        private readonly CMapSubTable? table;
+        private readonly CMapSubTable[] platformTables;
 
         public CMapTable(CMapSubTable[] tables)
         {
             this.Tables = tables;
 
             // lets just pick the best table for us.. lets jsut treat everything as windows and get the format 4 if possible
-            CMapSubTable? table = null;
+            var tbls = new List<CMapSubTable>();
             foreach (CMapSubTable t in this.Tables)
             {
                 if (t != null)
                 {
                     if (t.Platform == PlatformIDs.Windows)
                     {
-                        ushort format = table?.Format ?? 0;
-                        if (t.Format > format)
-                        {
-                            table = t;
-                        }
+                        tbls.Add(t);
                     }
                 }
             }
 
-            this.table = table;
+            this.platformTables = tbls.ToArray();
```

Comment:
Is there a reason we have to create a new array here?

Reply:
everytime we lookup a glyph from a code point (hot path) we have to iterate over the  `platformTables` collectino and i'm under the impression that `foreach`ing an `Array` is quicker than a `List<T>`

Reply:
I'd have to benchmark to be sure. Fine for now anyway....